content/blog/deploying-simple-jitsi-meet-server.md

```diff
@@ -0,0 +1,315 @@
+
+There are a variety of reasons
+why you may consider hosting Jitsi Meet yourself
+instead of relying on an already existing hosted instance:
+
+ * You stay in control of your own data.
+   When relying on a third-party service,
+   you have to trust them,
+   and in case you are using the service professionally,
+   you may be legally required to sign a contract on data processing with the provider.
+ * You can have your own branding.
+   Not only does the service run on your own domain,
+   but you can also customise the logo, welcome page, etc.
+ * You can configure it to your liking.
+   Depending on the typical use case of the instance,
+   some optional features might be desirable
+   that are not commonly found on other instances.
+ * You can update the service when you want.
+   If you need the service to work,
+   the user interface totally changing during an important meeting may be problematic.
+   By hosting it yourself, you can decide when to update it.
+
+### Why NixOS
+
+So why deploy this on NixOS and not on Debian or with Docker?
+NixOS has many advantages for running servers.
+It allows you to configure your whole server declaratively,
+so you can reuse the same configuration for multiple servers.
+Also, because it is reproducible,
+setting up a server with the same NixOS configuration
+will ensure that it runs the exact same software with the same configuration.
+Finally, if an update breaks your setup,
+NixOS makes rollbacks as easy as running one command or rebooting,
+so you can minimise downtime.
+
+## Requirements
+
+We will implement a small single-server deployment,
+so you need one server (e.g., a VPS).
+What its specifications should be,
+depends on how you want to use it.
+For just testing things out, one CPU core and 2 GB RAM should be enough,
+though you will run into limitations pretty quickly.
+The [official Jitsi Meet devops guide](https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-requirements) recommends 4 CPU cores and 8 GB RAM.
+If you don’t care about meeting recordings,
+a 15 GiB disk could be enough,
+though it doesn’t hurt to have more.
+Recordings, however, will dramatically increase the resource consumption.
+You will need at least 8 GB RAM, if possible even more,
+and a CPU fast enough to keep up with handling the meeting while simultaneously encoding it.
+Obviously, it also requires a larger disk to store the recordings on.
+
+You will also need a domain where Jitsi Meet should be available.
+
+If you already have a NixOS server set up
+that has the required resources available,
+you can skip to [Getting basic Jitsi Meet service running](#getting-basic-jitsi-meet-service-running).
+
+For configuring and deploying it,
+[Nix](https://nixos.org) with flakes support is required to be installed on your local system.
+If you have Nix version 3 or above, everything is already set up.
+Otherwise, you need to set the Nix option `experimental-features` to `nix-command flakes`.
```

Comment:
There should be a link to a canonical resource that says how to enable flakes. Right now the best we have is [NixOS wiki: Flakes](https://nixos.wiki/wiki/Flakes#Enable_flakes).